packages/ckeditor5-special-characters/src/ui/charactergridview.js

```diff
@@ -56,6 +58,18 @@ export default class CharacterGridView extends View {
 			}
 		} );
 
+		/**
+		 * Tracks information about the DOM focus in the grid.
+		 *
+		 * @readonly
+		 * @member {module:utils/focustracker~FocusTracker}
+		 */
+		this.focusTracker = new FocusTracker();
+
+		this.keystrokes = new KeystrokeHandler();
+
+		addKeyboardHandlingForGrid( this.keystrokes, this.focusTracker, this.tiles, 10 );
```

Comment:
What about the issue related to Windows vs Mac rendering? Can we hardcode the grid geometry in CSS to make it safer?

Reply:
Yes, we will handle it in [#9037](https://github.com/ckeditor/ckeditor5/issues/9037).